editor/templates/place/edit.html

```diff
@@ -19,6 +20,7 @@
 {{ super() }}
 <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js" integrity="sha512-/Nsx9X4HebavoBvEBuyp3I7od5tA0UzAxs+j83KgC8PU0kgB4XiK4Lfe4y4cgBtaRJQEIFCW+oC506aPT2L1zw==" crossorigin=""></script>
 <script src="https://unpkg.com/leaflet-ajax@2.1.0/dist/leaflet.ajax.min.js" integrity="sha384-sKs8ZrrxyJoElcPVznZwGpUTTXvkMYfHYxdIFzO8Hd0TA6emONMj8BwnsFf+6cZ/" crossorigin=""></script>
+<script src="https://unpkg.com/leaflet-draw/dist/leaflet.draw.js"></script>
```

Comment:
Should this have an integrity check like main Leaflet import? and crossorigin set?

Reply:
Resolved via https://github.com/iandees/wof-editor/pull/85/commits/65cd7b0ed6213e4747b1a66916d62f42c7d8cd36.